source/messaging/managing-channels.rst

```diff
@@ -154,4 +154,4 @@ Navigating recently viewed channels
 Use the **History** arrows at the top of the sidebar to move back and forth through your channel history. 
 
 - Select the left arrow to go back one page. 
-- Select the right arrow to go forward one page.
+- Select the right arrow to go forward one page. 
```

Comment:
```suggestion
- Select the right arrow to go forward one page.
```

Reply:
We don't use trailing white spaces in our documentation - but I am happy to discuss the reason behind this change if you're open to it?

Reply:
It was an error from my side. Sorry for the whitespace.